### PR TITLE
feat(web): set setting tab in fields modal as a default tab

### PR DESCRIPTION
**Pull request by issmail-basel**
## Solves
- FE: “settings” tab should be shown when model opens ( now the tab opened last time is opened )

### CODE REVIEW

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx [hunk 1]
```diff
@@ -83,9 +83,17 @@ const FieldUpdateModal: React.FC<Props> = ({
...
   const { TabPane } = Tabs;
   const selectedValues: string[] = Form.useWatch("values", form);
 
+  const changeTab = useCallback(
+    (key: string) => {
+      setActiveTab(key as FieldModalTabs);
+    },
+    [setActiveTab],
+  );
+
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx:96]:
```suggestion
  const changeTab = useCallback(
    (tab: FieldModalTabs) => {
      setActiveTab(tab);
    },
    [setActiveTab],
  );

```

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx [hunk 2]
```diff
@@ -154,18 +162,17 @@ const FieldUpdateModal: React.FC<Props> = ({
...
-        form.resetFields();
         onClose?.(true);
       })
       .catch(info => {
         console.log("Validate Failed:", info);
       });
   }, [form, onClose, onSubmit, selectedType, selectedField?.id]);
 
-  const handleClose = useCallback(() => {
+  const resetModal = useCallback(() => {
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx:172]: handleModalReset

File: web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx [hunk 3]
```diff
@@ -129,18 +137,17 @@ const FieldCreationModal: React.FC<Props> = ({
...
-        form.resetFields();
         onClose?.(true);
       })
       .catch(info => {
         console.log("Validate Failed:", info);
       });
   }, [form, onClose, onSubmit, selectedType]);
 
-  const handleClose = useCallback(() => {
+  const resetModal = useCallback(() => {
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx:147]: handleModalReset

Comment (issmail-basel) [reply web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx]: @KaWaite , fixed.
https://github.com/reearth/reearth-cms/blob/a78d0940f3829779de4c52fec808ac0823e29680/web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx#L147

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx (hunk 2 above)
Comment (issmail-basel) [reply web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx]: @KaWaite , fixed.
https://github.com/reearth/reearth-cms/blob/a78d0940f3829779de4c52fec808ac0823e29680/web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx#L172

File: web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx [hunk 4]
```diff
@@ -80,9 +80,17 @@ const FieldCreationModal: React.FC<Props> = ({
   const t = useT();
   const [form] = Form.useForm();
   const [buttonDisabled, setButtonDisabled] = useState(true);
+  const [activeTab, setActiveTab] = useState<FieldModalTabs>("settings");
   const { TabPane } = Tabs;
   const selectedValues: string[] = Form.useWatch("values", form);
 
+  const changeTab = useCallback(
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx:87]: HandleTabChange

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx [hunk 5]
```diff
@@ -83,9 +83,17 @@ const FieldUpdateModal: React.FC<Props> = ({
   const t = useT();
   const [form] = Form.useForm();
   const [buttonDisabled, setButtonDisabled] = useState(false);
+  const [activeTab, setActiveTab] = useState<FieldModalTabs>("settings");
   const { TabPane } = Tabs;
   const selectedValues: string[] = Form.useWatch("values", form);
 
+  const changeTab = useCallback(
```
Comment (KaWaite) [inline web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx:90]: HandleTabChange

**Review by KaWaite — APPROVED** [review]

File: web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx (hunk 4 above)
Comment (issmail-basel) [reply web/src/components/molecules/Schema/FieldModal/FieldCreationModal/index.tsx:87]: @KaWaite . fixed.

File: web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx (hunk 5 above)
Comment (issmail-basel) [reply web/src/components/molecules/Schema/FieldModal/FieldUpdateModal/index.tsx:90]: @KaWaite . fixed.

**Pull request merged**